sphinx_doc/howtos/tutorials/docs/navigation2_on_real_turtlebot3.rst

```diff
@@ -0,0 +1,161 @@
+.. _navigation2-on-real-turtlebot3:
+
+Navigation2 on a Real Turtlebot 3
+*********************************
+
+- `Overview`_
+- `Requirements`_
+- `Tutorial Steps`_
+- `Videos`_
+
+Overview
+========
+
+This tutorial shows how to control and navigate Turtlebot 3 using the ROS2 Navigation2 on a real Turtlebot 3 robot.
+Before completing this tutorials, completing the :ref:`navigation2-with-turtlebot3-in-gazebo`. is highly recommended especially if you are new to ROS and Navigation2.
+
+Turtlebot is a low-cost, personal robot kit with open-source software (ROS2).
+Turtblebot robots are widely supported by the ROS community.
+You can find many applications and examples of Turtlebot projects on the Internet.
+You can find more information about Turtlebot3 `here. <http://emanual.robotis.com/docs/en/platform/turtlebot3/overview/>`_
+
+Links to the robot kits and Github repo.
```

Comment:
If people have gone through the gazebo tutorial this is all redundant and not entirely on topic. Please delete lines 17 through 26

Reply:
to be honest I didn't read the tutorial. I just converted it to rst format.